metrics/storage/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Set up an Ubuntu image with 'blogbench' installed
+
+# Usage: FROM [image name]
+FROM ubuntu
+
+RUN apt-get update && \
+	apt-get install -y build-essential wget && \
+	wget https://download.pureftpd.org/pub/blogbench/blogbench-1.1.tar.gz && \
+	tar xvzf blogbench-1.1.tar.gz && \
```

Comment:
Version hard-coded multiple times. The ideal would be to have this version in a `versions.txt` or `versions.yaml` and create this `Dockerfile` from `Dockerfile.in`. Alternatively, I'd create a variable in this file to avoid having to duplicate it.

Reply:
Valid point @jodh-intel I will note that the blogbench test has not been modified since 2015 though... ;-)
We may as well add the env var and do the edits though @GabyCT to remove the duplication.